mappings/net/minecraft/inventory/InventoryChangedListener.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_1265 net/minecraft/inventory/InventoryChangedListener
```

Comment:
`InventoryListener` is better IMO (or `InventoryChangeListener` without the D).

Reply:
`InventoryCallback`

Reply:
It only receives events for when an inventory's contents change. It's not a general listener for inventories, otherwise it would have `onOpen` and `onClose`.

Keeping the `Change` part of the class name.